evap/evaluation/views.py

```diff
@@ -198,29 +198,34 @@ def set_lang(request):
 
 def profile_edit(request):
     user = request.user
-    if user.is_editor:
-        form = DelegatesForm(request.POST or None, request.FILES or None, instance=user)
-
-        if form.is_valid():
-            form.save()
-
-            messages.success(request, _("Successfully updated your profile."))
-            return redirect("evaluation:profile_edit")
-
-        return render(
-            request,
-            "profile.html",
-            {
-                "user": user,
-                "form": form,
-                "delegate_of": user.represented_users.all(),
-                "cc_users": user.cc_users.all(),
-                "ccing_users": user.ccing_users.all(),
-            },
-        )
+    profile_form = ProfileForm(request.POST or None, request.FILES or None, instance=user)
+    form = DelegatesForm(request.POST or None, request.FILES or None, instance=user)
+
+    if request.method == "POST":
+        operation = request.POST.get("operation")
+        if operation not in ("profile", "delegates"):
+            raise SuspiciousOperation("Invalid POST operation")
```

Comment:
Actually, I think it would be better to create two new views for the two forms on the profile site, maybe `/profile/edit` and `/profile/request_changes`. This way, we get rid of the `operation` parameter :)

Reply:
@niklasmohrin I did not make this into two views for now, as discussed offline, it's a bit annoying to do. Resolving this discussion for now. Feel free to open another issue for that (would then also affect staff import persons view).